main.go

```diff
@@ -50,8 +50,10 @@ func parserTest() {
 		fmt.Println("parsing devfile from " + os.Args[1])
 
 	} else {
+		trueFlag := true
 		args = parser.ParserArgs{
-			Path: "devfile.yaml",
+			Path:                             "devfile.yaml",
+			ConvertDockerComposeToKubernetes: &trueFlag, //setting flag to true for parser
```

Comment:
You could use the [`k8s.io/utils/pointer`](https://pkg.go.dev/k8s.io/utils/pointer) or [`k8s.io/utils/ptr`](https://pkg.go.dev/k8s.io/utils/ptr) package for this, e.g.

```suggestion
			ConvertDockerComposeToKubernetes: pointer.Bool(true), //setting flag to true for parser
```